ParabolInc/parabol
# refactor: gitlab issues query part 2

Fixes https://github.com/ParabolInc/parabol/issues/6325

Predecessor PR: https://github.com/ParabolInc/parabol/pull/6563

### To test

- Integrate with GitLab and go to the Scope phase of a Poker meeting
- Enter a search query and create a new issue. See that the newly create issue is rendered at the top of the list
- Select a project filter and create a new issue. See that the newly create issue is rendered at the top of the list
- Don't select any project filters, scroll down, and see that the list paginates
- Select one project filter, scroll down, and see that the list paginates
- Select multiple project filters, scroll down, and see that the list paginates showing 25 more issues for both projects
